.gitignore

```diff
@@ -33,3 +33,6 @@ nosetests.xml
 .mr.developer.cfg
 .project
 .pydevproject
+
+# Pycharm
+.idea/
```

Comment:
I believe you can but there are quite a few general use cases when you might want to keep the .idea config files as well as others

Reply:
I know this is a silly hill to die on, but I won't add gitignore patterns for every editor under the sun. Unless I'm clearly overruled by community consensus.

The Git documentation has [many alternatives](https://git-scm.com/docs/gitignore) to ignore files. Since this pattern is not relevant to other gcovr developers, consider using `.git/info/exclude` instead.